docs/source/ops/connect/helm.rst

```diff
@@ -0,0 +1,281 @@
+and a separate PostgreSQL instance will be started for the Trigger Service. See
+the end of this page for details.
+
+Production Setup
+----------------
+
+There are many options you may want to set for a production setup. See the
+reference at the end of this page for full details. At a minimum, though, you
+need to set the following:
+
+- ``production=true``: By default, the Helm chart starts a number of components
+  that are meant to give you a quick idea of what the Helm chart enables, but
+  are most definitely not meant for production use. Specifically, this will
+  disabble the internal PostgreSQL instances, the mock auth server, and the
+  reverse proxy.
+- ``ledger.db``: If you want the Helm char to start a Daml Driver For
+  PostgreSQL instance for you, you need to set this. See reference section at
+  the end of this page for details.
+- ``ledger.host`` and ``ledger.port``: If you **do not** want the Helm chart to
+  setup a Daml Driver isntance for you, but instead want the components started
+  by it to connect to an existing Ledger API server, fill in these options
+  instead of the ``ledger.db`` object.
+- ``jsonApi.db``: If you want the Helm chart to start the HTTP JSON API Service
+  for you, you need to set this. See reference section at the end of this page
+  for details.
+- ``triggerService.db``: If you want the Helm chart to start the Trigger
+  Service for you, you need to set this. See reference section at the end of
+  this page for details.
+- ``authUrl``: If you want the Helm chart to provide either a Daml Driver for
+  PostgreSQL or a OAuth2 Middleware instance, you will need to set this to the
+  JWKS URL of your token provider.
+
+If you start the Trigger Service, you will need to configure it, as well as the
+OAuth2 Middleware. See the required options for them in the reference section
+at the end of this page.
+
+Finally, we also recommend looking at the ``resources`` option for each
+component and adjusting them to fit your particular use-case.
+
+Log Aggregation
+---------------
+
+All processes write their logs directly to stdout. This means that log
+aggregation can be addressed at the Kubernetes level and does not require any
+specific support from the Helm chart itself. One fairly easy way to achieve
+this is using `Filebeat <https://www.elastic.co/beats/filebeat>`_, which
+regulary collects the logs of your containers and ingests them into
+`Elasticsearch <https://www.elastic.co/elasticsearch/>`,
+`Logstash <https://www.elastic.co/logstash/>`_,
+`Kafka <https://kafka.apache.org/>`_, etc.
+
+You can find external documentation on, how to setup `ElasticSearch` with
+`Filebeat` and `Kibana` for analyzing logs on your Kubernetes cluster
+`here <https://www.deepnetwork.com/blog/2020/01/27/ELK-stack-filebeat-k8s-deployment.html>`_.
+
+As of 1.18.0, the :ref:`HTTP JSON API <json-api>` component in the Helm chart produces
+JSON-encoded logs. Other components log in somewhat less structured formats.
```

Comment:
```suggestion
JSON-encoded logs. Other components log as unstructured text.
```

Reply:
That was my first version, but it's not actually completely unstructured, is it? Anyway, done.